app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -257,7 +267,7 @@ class MetadataManualInput extends React.Component {
   isHostGenomeIdValidForField = (hostGenomeId, field) =>
     includes(
       hostGenomeId,
-      this.props.projectMetadataFields[field].host_genome_ids
+      get([field, "host_genome_ids"], this.props.projectMetadataFields) || false
```

Comment:
@MarkAZhang I was getting an "undefined" error here when I tried "apply all" . This seems like an appropriate fix. Please confirm!

Reply:
Hey, this is definitely safer and should be correct.

I don't think you need the `|| false`. `get(...)` will default to undefined, and `includes` should be able to accept `undefined` as a parameter and default gracefully to `false`. One of the great things about lodash.

I'm curious how you got this error! I wasn't able to reproduce it when I tried briefly.

Reply:
Thanks. I'll remove the `false`. I got the error using the test files someone shared with the team around month ago. 

![image](https://user-images.githubusercontent.com/28797/56605293-abaadc00-65b8-11e9-8ade-25a256d5f032.png)